### PR TITLE
docs: improve grpc third party client documentation

### PR DESCRIPTION
**Pull request by samsja**
# Context

Right now the docs to use grpc from an other language are really shallow. 

docs to check for review : 
* https://docs-improve-client--jina-docs.netlify.app/fundamentals/client/client/ 
* https://docs-improve-client--jina-docs.netlify.app/fundamentals/client/third-party-client/

## What this pr do

- [x] explain how to communicate to the gateway with gRPC outside a python in a abstract way
- [ ] TODO LATER add an example with an external language

### CODE REVIEW

File: docs/fundamentals/client/client.md [hunk 1]
```diff
@@ -2,10 +2,16 @@
...
 {class}`~jina.Client` enables you to send Documents to a running {class}`~jina.Flow` in a number of different ways, as shown below.
 
-Clients support three different networking protocols: HTTP, gRPC, WebSocket and GraphQL
+Clients support four different networking protocols: **HTTP**, **gRPC**, **WebSocket** and **GraphQL**
 
 For each of them, you first connect your Client to the API Gateway, before you can send requests to it.
 
+```{hint}
+You want to connect to your Flow from an other programming language than Python ? Please follow the third party 
+client {ref}`documentation <third-party-client>`
```
Comment (JohannesMessner) [inline docs/fundamentals/client/client.md:11]:
```suggestion
If you want to connect to your Flow from a programming language other than Python, please follow the third party 
client {ref}`documentation <third-party-client>`.
```

File: docs/fundamentals/client/third-party-client.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
-(access-flow-api)=
+(third-party-client)=
 # Third-party clients
 
-This page is about accessing the Flow with other clients, e.g. `curl`.
+This page is about accessing the Flow with other clients, e.g. `curl` and other programming language than python.
```
Comment (JohannesMessner) [inline docs/fundamentals/client/third-party-client.md:4]:
```suggestion
This page is about accessing the Flow with other clients, e.g. `curl`, or programming languages other than python.
```

File: docs/fundamentals/client/third-party-client.md [hunk 3]
```diff
@@ -230,6 +230,17 @@ We provide a suite of templates for the Jina Flow, in this [collection](https://
 
 This contribution was made by [Jonathan Rowley](https://jina-ai.slack.com/archives/C0169V26ATY/p1649689443888779?thread_ts=1649428823.420879&cid=C0169V26ATY), in our [community Slack](slack.jina.ai). 
 
+## gRPC
+
+To use the gRPC protocol with another language that python you will need to :
```
Comment (JohannesMessner) [inline docs/fundamentals/client/third-party-client.md:235]:
```suggestion
To use the gRPC protocol with a language other than Python you will need to :
```

File: docs/fundamentals/client/third-party-client.md [hunk 4]
```diff
@@ -230,6 +230,17 @@ We provide a suite of templates for the Jina Flow, in this [collection](https://
...
+
+* Download the two proto definition files : `jina.proto` and `docarray.proto` from [github](https://github.com/jina-ai/jina/tree/master/jina/proto) (be sure to use the latest release branch)
```
Comment (JohannesMessner) [inline docs/fundamentals/client/third-party-client.md:237]:
```suggestion
* Download the two proto definition files: `jina.proto` and `docarray.proto` from [github](https://github.com/jina-ai/jina/tree/master/jina/proto) (be sure to use the latest release branch)
```

File: docs/fundamentals/client/third-party-client.md [hunk 5]
```diff
@@ -230,6 +230,17 @@ We provide a suite of templates for the Jina Flow, in this [collection](https://
...
+* Compile them with [protoc](https://grpc.io/docs/protoc-installation/) and precise to which programming language you want to compile them.
+* Add the generated files to your project and import them in your code. 
+
+You should finally be able to communicate with your Flow using the gRPC protocol. You can find more information on the gRPC
+`message` and `service` that you can use to communicate in the  [Protobuf documentation](../../proto/docs.md).
```
Comment (JohannesMessner) [inline docs/fundamentals/client/third-party-client.md:242]: What is this code that they will compile and add to their project? stubs that they can call directly? should we mention how this works or is that outside of our scope here?

**Review by JohannesMessner — CHANGES_REQUESTED** [review]
Some suggestions

File: docs/fundamentals/client/third-party-client.md (hunk 5 above)
Comment (samsja) [reply docs/fundamentals/client/third-party-client.md:242]: It is out of the scope I believe. The only way to do it correctly is to 
* have a good proto documentation which explain the stubs that should be use etcc
* have a good how to with another language

**Review by JohannesMessner — APPROVED** [review]
LGTM

**Pull request merged**